### ISSUE TITLE
Implement quota manager

### ISSUE THREAD

**Issue by yaronyg** [thaliproject/Thali_CordovaPlugin#419]
Depends on #319

**Comment by artemjackson** [comment]
Specification updated. See [#319](https://github.com/thaliproject/Thali_CordovaPlugin/issues/319#issuecomment-239441949) for more details.

**Comment by yaronyg** [comment]
Closing in favor of the PR
